docs/handbook/strategy.md

```diff
@@ -66,6 +66,67 @@ What we will build:
 
 Making money:
 
-* We will focus on small customers first. This is because we will optimize for learning. There are more of them and they will be faster to learn from. Product analytics tools often charge 100 person startups $20-70K/year.
-* We can undercut SAAS product analytics tools because we have no hosting costs.
-* Over time, we will put more work into our innovation, and the biggest and brightest companies will realize that we are the market leader.
+We'll have three routes to generating revenue
+
+* Set up a self-serve hosted version
+* Build some of the growth/ultimate plan features like experimentation
+* Support self-hosted
```

Comment:
I think it's going to be more important in the immediate future to get way more companies using it than to try to get money out of everyone. This is how every massive open source company has grown. If we can just make the free version developer experience outstanding, we'll get a massive amount of organic growth and inbound interest. That's the most important thing of all.

However, we should have a clear view of how revenue will work, so we don't introduce free features that we later want to make paid since that'll cause friction. As a minimum, I'd suggest having some way of at least capturing paid demand would help us get this right.

It feels that charging for hosting would be an easy win, and setting up a preorder form (in exchange for a discount) for the paid functionality in the immediate future would give us a signal for the level of demand at least.

Reply:
I would like to second the point about switching free features to paid versions - so far the issues raised by the community that someone at PostHog comments on as a likely paid feature feels enough on the signal front, those users are clearly incentivized to argue why it should be free but in turn allows us to put forward a defensible reason as to why it should be part of growth. 

If we feel our "growth feature" is aligned well with a customer issue we should place that feature (and others like it) in "features" in our docs clearly under growth, when new issues are raised in line with this we can then easily count the interest in that feature along with a clear explanation of why it is part of growth or ultimate. 

Because revenue generation is not prioritized above growing usage within the community version it doesn't matter if these are being raised every week, or the total number - it feels like at the current rate of discussion with new issues that we will have enough to fit that bucket come the time we have finished the parity project and developer features project.

Reply:
Updated